modules/social_features/social_private_message/src/Plugin/Field/FieldFormatter/SocialPrivateMessageThreadMemberFormatter.php

```diff
@@ -27,79 +21,10 @@
  */
 class SocialPrivateMessageThreadMemberFormatter extends PrivateMessageThreadMemberFormatter {
 
-  /**
-   * Renderer services.
-   *
-   * @var \Drupal\Core\Render\Renderer
-   */
-  protected Renderer $renderer;
-
-  /**
-   * Construct a PrivateMessageThreadFormatter object.
-   *
-   * @param string $plugin_id
-   *   The ID of the plugin.
-   * @param mixed $plugin_definition
-   *   The plugin definition.
-   * @param \Drupal\Core\Field\FieldDefinitionInterface $field_definition
-   *   The field definition.
-   * @param array $settings
-   *   The field settings.
-   * @param mixed $label
-   *   The label of the field.
-   * @param string $view_mode
-   *   The current view mode.
-   * @param array $third_party_settings
-   *   The third party settings.
-   * @param \Drupal\Core\Entity\EntityTypeManagerInterface $entityTypeManager
-   *   The entity manager service.
-   * @param \Drupal\Core\Session\AccountProxyInterface $currentUser
-   *   The current user.
-   * @param \Drupal\Core\Entity\EntityDisplayRepositoryInterface $entity_display_repository
-   *   The entity display repository.
-   * @param \Drupal\Core\Render\Renderer $renderer
-   *   The renderer service.
-   */
-  public function __construct(
-    $plugin_id,
-    $plugin_definition,
-    FieldDefinitionInterface $field_definition,
-    array $settings,
-    $label,
-    $view_mode,
-    array $third_party_settings,
-    EntityTypeManagerInterface $entityTypeManager,
-    AccountProxyInterface $currentUser,
-    EntityDisplayRepositoryInterface $entity_display_repository,
-    Renderer $renderer
-  ) {
-    parent::__construct($plugin_id, $plugin_definition, $field_definition, $settings, $label, $view_mode, $third_party_settings, $entityTypeManager, $currentUser, $entity_display_repository);
-    $this->renderer = $renderer;
-  }
-
-  /**
-   * {@inheritdoc}
-   */
-  public static function create(ContainerInterface $container, array $configuration, $plugin_id, $plugin_definition) {
-    return new static(
-      $plugin_id,
-      $plugin_definition,
-      $configuration['field_definition'],
-      $configuration['settings'],
-      $configuration['label'],
-      $configuration['view_mode'],
-      $configuration['third_party_settings'],
-      $container->get('entity_type.manager'),
-      $container->get('current_user'),
-      $container->get('entity_display.repository'),
-      $container->get('renderer'),
-    );
-  }
-
   /**
    * {@inheritdoc}
    */
-  public function viewElements(FieldItemListInterface $items, $langcode) {
```

Comment:
FYI - Because the parent is now the same we don't need to do this anymore see:
https://git.drupalcode.org/project/private_message/-/blob/8cd60c5124411527475604c044786c93fa45c432/src/Plugin/Field/FieldFormatter/PrivateMessageThreadMemberFormatter.php